package.json

```diff
@@ -44,7 +49,22 @@
     "eslint-plugin-react-native": "^4.1.0",
     "eslint-plugin-yml": "^1.12.2",
     "husky": "^9.0.11",
+    "jest-expo": "^50.0.4",
```

Comment:
Was this automatically added as a dev dependency?

Reply:
I belive so yes. I followed these instructions: https://docs.expo.dev/develop/unit-testing/. Should it be only in normal dependencies ?

Reply:
It is possible that it is needed as a dev. dependency too. You can check by removing it from the dev. dependencies list and then doing an `npm install`. If the tests still execute without issues, it means that the package is not needed as a dev. dependency.

Reply:
Ok, works fine wihtout it. Removed it.